bundles/org.openhab.binding.boschshc/README.md

```diff
@@ -8,8 +8,8 @@ Binding for Bosch Smart Home devices.
     - [In-Wall Switch](#in-wall-switch)
     - [Compact Smart Plug](#compact-smart-plug)
     - [Twinguard Smoke Detector](#twinguard-smoke-detector)
-    - [Door/Window Contact](#door-window-contact)
-    - [Door/Window Contact II](#door-window-contact-ii)
+    - [Door/Window Contact](#doorwindow-contact)
```

Comment:
I don't think these new link targets are correct. At least the code completion in the WikiText editor proposes `door-window-contact` here and with that the links are working. If I change to `doorwindow-*`, I get validation warnings and the links do not work anymore:

<img width="449" height="33" alt="grafik" src="https://github.com/user-attachments/assets/9602cf01-365d-4a24-ae10-f4633eb80ff4" />

Reply:
Some editors can probably not cope well with the slash in the title. VSCode accepts this and in the preview mode it is clickable. The markdownlint-cli tool also accepts this while the previous link was showing a warning.

Any suggestions? Rename the title to make it work for all ide's?

Reply:
Unfortunately there is no standard. For now it is important that it fits in the toling: VSCode/Eclipse, Github and openHAB doc rendering. So i think we are good.

Reply:
`markdownlint-cli` has been instructed to make use of the configuration listed at the openHAB site:
https://github.com/openhab/openhab-docs/blob/main/.github/.markdownlint.yaml I have configured my IDE (VSCode) wioth the same markdown tool and configuration file.

So i'm prettu sure it will work and if not there will be a bug and time (as this will be part of openHAB 5.1) to fix it. 
I guess the parser that is used to generate the docs pages is able to coop with different formats.

Reply:
ok, let's merge it like this, but please check the resulting final HTML output if the TOC link and the anchor are in sync, or let me know where/how I could verify it.

Reply:
@lsiepel the links at https://next.openhab.org/addons/bindings/boschshc/ are now broken.

Example Link: https://next.openhab.org/addons/bindings/boschshc/#doorwindow-contact

Actual Link Target: https://next.openhab.org/addons/bindings/boschshc/#door-window-contact

Should we change the Markdown back or do you want to fix this on the HTML generation side?

Reply:
FYI @david-pace https://github.com/openhab/openhab-addons/pull/19145